erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md

```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+# E Invoicing under GST
+
+Under GST, all small and large businesses are supposed to register all sales invoices onto GST portal that falls under one of the following categories:
+- Business to Business
+- Supply to SEZ
+- Exports
+- Deemed Exports
+
+To help automate the E Invoicing process, we have integrated ERPNext with a GSP so you can easily generate IRNs and signed E Invoices from your ERPNext Sales Invoices.
+
+## Prerequisites
+
+- You must have a registered account on E Invoice [portal](https://einvoice1.gst.gov.in/)
+- GST Accounts must be set in the **GST Settings** DocType.
+
+## Setting up
+
+
+### Getting Credentials
+
+1. Login into E Invoice [portal](https://einvoice1.gst.gov.in/) with your username & password.
+1. Click on **API registration** on the sidebar.
+1. Click on **User Credentials** from the expanded list.
+1. Click on **Create API User**.
+1. Click on **Through GSP** and select **Adequare Info Private Limited**.
+1. Create a username and password which will be used to authenticate with GSP.
+
+### Setting Up ERPNext
+
+Go to **E Invoice Settings** and click on **Enable** checkbox.
+
+1. **GSTIN**: GSTIN by which your company is registered on e-invoice portal.
+1. **Username**: Username created in previous step.
+1. **Password**: Password created in previous step.
+
+### Generating IRN
+
+Create a sales invoice and keep it under Draft state. Click on **E Invoicing** button group and then on **Generate IRN**. If Sales Invoice doesn't have any validation errors, IRN will be generated and updated in the Sales Invoice. You can now submit the invoice and print E Invoice with QRCode image by selecting **GST E Invoice** Print Format while printing.
+
+<img class="screenshot" src="/docs/assets/img/regional/india/einv_gen_irn_button.png">
```

Comment:
3 images with no context, add sentences in between saying what the images are about

Reply:
Done